### ISSUE TITLE
single & multi-select drop down list

### ISSUE THREAD

**Issue by gecon27** [solidify/jira-azuredevops-migrator#300]
Refer to https://github.com/solidify/jira-azuredevops-migrator/issues/40#issuecomment-482722481 and previous comment. I haven't seen any further actions on this topic. What I'd like to see is single & multi-select drop down lists, similar to https://marketplace.visualstudio.com/items?itemName=ms-devlabs.vsts-extensions-multivalue-control. Thanks for the good work up to now.

**Comment by RoyalWulf** [comment]
Hi, Would like this feature, any progress? Cheers

**Comment by AmarnathReddyN** [comment]
Any update on the multi-select drop down list

**Comment by Alexander-Hjelm** [comment]
No, and we will probably not prioritize functionality outside of the Core ADO features for now :)

**Comment by AmarnathReddyN** [comment]
If it's not planning now, is there any option of  customizing it by own. 
Pls share thought of how we can customize by self to own project, as in my project migration the multi select field is must to migrate it.

**Comment by Alexander-Hjelm** [comment]
Migrate the raw data to a text field and then bulk copy (+transform) the contents of that field to your custom field? You may need a custom script solution if you cannot use ADO Queries
